lib/plugins/aws/info/index.js

```diff
@@ -62,6 +63,8 @@ class AwsInfo {
 
       'aws:info:displayStackOutputs': async () =>
         BbPromise.bind(this).then(this.displayStackOutputs),
+
+      'aws:info:displayServiceOutputs': async () => await this.displayServiceOutputs(),
```

Comment:
For now I'm keeping it here so it's still going to be printed before the finalization happens - I'll move it to proper place in separate PR